helm-charts/basehub/values.yaml

```
@@ -409,3 +414,8 @@ jupyterhub:
             if authenticator_class == "github" and c.Authenticator.allowed_users:
                 print("WARNING: hub.config.JupyterHub.authenticator_class was set to github and c.Authenticator.allowed_users was set, custom 2i2c jupyterhub config is now resetting allowed_users to an empty set.")
                 c.Authenticator.allowed_users = set()
+      08-add-docs-service-if-enabled: |
+        from z2jh import get_config
+
+        if get_config("custom.docs_service.enabled"):
+          c.JupyterHub.services.append({"name": "docs", "url": "http://docs-service"})
```

I don't think its an error to use two spaces to indent in Python, but let's use four consistently.

```suggestion
            c.JupyterHub.services.append({"name": "docs", "url": "http://docs-service"})
```

---

The thing is that [most of the code](https://github.com/2i2c-org/infrastructure/blob/master/helm-charts/basehub/values.yaml#L328) in `hub.extraConfig` is indented using two spaces, only a few use four. But agree, there should be consistency there. Do you prefer to change it all to four or two?

---

Ah, I suggest we ignore this for now then and in a standalone PR for the future maybe go make all of them indent like python code almost always is indented - with four spaces.